WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -1044,6 +1063,18 @@ extension OrderDetailsDataSource {
                            headerStyle: headerStyle)
         }()
 
+        let customFields: Section? = {
+            guard featureFlags.isFeatureFlagEnabled(.orderCustomFields) else {
+                return nil
+            }
+
+            guard order.customFields.isNotEmpty else {
+                return nil
+            }
```

Comment:
nit: We can combine these two `guard` statements to be more concise:

```suggestion
            guard featureFlags.isFeatureFlagEnabled(.orderCustomFields), order.customFields.isNotEmpty else {
                return nil
            }
```

Reply:
Nice suggestion! Done here: https://github.com/woocommerce/woocommerce-ios/pull/7257/commits/c4ab53f5f944f162090d64b632e29d693d0be976